TheOracle2/ProgressTrack/Interfaces/ILogWidget.cs

```diff
@@ -0,0 +1,34 @@
+namespace TheOracle2.GameObjects;
```

Comment:
I like where this is going, but I think it should probably use the event pattern, and remove most of the AlterOn stuff. I have some stuff planned for this space, so lets leave this for now, and revisit it in the upcoming player logs PR.

Reply:
sounds good - it's pretty skeletal and i don't think its method is presently use, but i wanted to reserve some space for it on certain interfaces/classes

Reply:
> I like where this is going, but I think it should probably use the event pattern, and remove most of the AlterOn stuff. I have some stuff planned for this space, so lets leave this for now, and revisit it in the upcoming player logs PR.

sounds good - it's pretty skeletal and i don't think its method is presently use, but i wanted to reserve some space for it on certain interfaces/classes